can/can_class_setting_status.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef HAL_EXTENSION_CAN_CLASS_SETTING_STATUS_HPP
+#define HAL_EXTENSION_CAN_CLASS_SETTING_STATUS_HPP
+
+#include "main.h"
+
+namespace halex {
+
+enum class CAN_ClassSettingStatus: uint8_t {
+    WRONG_SET_PATH_ID_FUNCTION_USED = 0x20,
+    ALL_ID_PATH                     = 0x21,
+    NON_ERROR                       = 0x00
+};
```

Comment:
```suggestion
enum class CAN_ClassSettingStatus: uint8_t {
    OK                              = 0x00,
    WRONG_SET_PATH_ID_FUNCTION_USED = 0x01,
    ALL_ID_PATH                     = 0x02
};
```

[WANT] 成功は0にして一番上に定義したいです。stm32_hal に合わせて OK を使いたいです。数字は連番でいいと思います

Reply:
4d14ed47d43aa0a264d42a0aede367aca409d104 で削除